src/runtime/runtime_tinygowasmp2.go

```diff
@@ -0,0 +1,76 @@
+//go:build wasip2
+
```

Comment:
What about merging this file and runtime_wasm_wasip2.go?
Or was there a reason you kept them separate?

Reply:
Probably just an artifact of how the port was initially written.  We can do a bit more cleanup later.